### PR TITLE
Add MousePosition control config

### PR DESCRIPTION
**Pull request by tonio**
- Add mouseposition control
 - Make styles closer to main app
 - Add logo

### CODE REVIEW

**Comment by pgiraud** [comment]
I rebase jsapi onto master. Can you please rebase the branch corresponding to the current pull request?

**Comment by tonio** [comment]
Rebased & ready for review.

File: jsapi/externs/luxx.js [hunk 1]
```diff
@@ -56,6 +58,20 @@ luxx.MapOptions.prototype.positionSrs;
 luxx.MapOptions.prototype.zoom;
 
 /**
+ * Set the presence of a mouse position control in the map. (not included by
+ * default).
+ * @type {ol.Coordinate|undefined}
```
Comment (pgiraud) [inline jsapi/externs/luxx.js]: {boolean|undefined} ?

Comment (tonio) [inline jsapi/externs/luxx.js]: fixed.

**Comment by tonio** [comment]
Rebased, travis OK

**Comment by pgiraud** [comment]
Looks good to me. Please merge.

**Pull request merged**